fbw-a380x/src/systems/instruments/src/MFD/pages/common/DataEntryFormats.tsx

```diff
@@ -435,7 +435,16 @@ export class TemperatureFormat implements DataEntryFormat<number> {
       return null;
     }
 
-    const nbr = Number(input);
+    let nbr = Number(input);
+    
+    if (nbr > 0 && input.substring(0, 1) !== '+') {
+      nbr *= -1;
+    }
+    
+    if (nbr > 0 && input.substring(0, 1) === '+') {
+      nbr = input.substring(1);
```

Comment:
I don't understand what this is trying to do. At this point nbr has already successfully been parsed, and you're now replacing it with a string (without parsing).